spec/dummy/config/routes.rb

```diff
@@ -1,7 +1,7 @@
 Dummy::Application.routes.draw do
   resources :widgets, only: :index
-  match '/widgets/bubble_up' => 'widgets#bubble_up'
-  match '/widgets/deep_bubble_up' => 'widgets#deep_bubble_up'
-  match '/widgets/catch' => 'widgets#catch'
-  match '/widgets/deep_catch' => 'widgets#deep_catch'
+  get '/widgets/bubble_up' => 'widgets#bubble_up'
+  get '/widgets/deep_bubble_up' => 'widgets#deep_bubble_up'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.